packages/react-dom/src/events/ReactDOMEventListener.js

```diff
@@ -11,6 +11,7 @@ import type {AnyNativeEvent} from '../events/PluginModuleType';
 import type {FiberRoot} from 'react-reconciler/src/ReactInternalTypes';
 import type {Container, SuspenseInstance} from '../client/ReactDOMHostConfig';
 import type {DOMEventName} from '../events/DOMEventNames';
+import type {EventPriority} from 'shared/ReactTypes';
```

Comment:
We're going to replace this with lane priority later. Conveniently, this file already imports them.